TextFormat.md

```diff
@@ -1,64 +1,76 @@
 # Text Format
 
-The purpose of this text format is to support:
-
-* View Source on a WebAssembly module, thus fitting into the Web (where every
-  source can be viewed) in a natural way.
-* Presentation in browser development tools when source maps aren't present
-  (which is necessarily the case with [the Minimum Viable Product (MVP)](MVP.md)).
-* Writing WebAssembly code directly for reasons including pedagogical,
-  experimental, debugging, optimization, and testing of the spec itself.
-
-The text format is equivalent and isomorphic to the [binary format](BinaryEncoding.md).
```

Comment:
Given that this was already merged, can you create a new PR with a clarifying statement? Happy to merge immediately--I didn't intend to remove that consideration.